src/Files.App/Views/Properties/SecurityPage.xaml

```diff
@@ -60,36 +67,30 @@
 						Orientation="Horizontal"
 						Spacing="8">
 
-						<Button
-							Padding="8"
-							Background="Transparent"
-							BorderBrush="Transparent"
-							Command="{x:Bind SecurityViewModel.AddAccessControlEntryCommand}">
+						<Button Padding="8" Command="{x:Bind SecurityViewModel.AddAccessControlEntryCommand}">
```

Comment:
Can you share a screenshot of the before and after?

Reply:
Before/After
![image](https://user-images.githubusercontent.com/62196528/232813180-2b4c640e-348d-4c57-b9cd-56869b547ba2.png)

Reply:
Can you revert this change?

Reply:
There are a lot of options there and the text and background "steal" the focus.

Reply:
Reverted